ibm_cloud_sdk_core/base_service.py

```diff
@@ -118,10 +119,10 @@ def configure_service(self, service_name: str):
                 bool(config.get('DISABLE_SSL'))
             )
 
-    def _set_user_agent_header(self, user_agent_string=None):
+    def _set_user_agent_header(self, user_agent_string=None) -> None:
```

Comment:
Missing type annotation on `user_agent_string` parameter.
```suggestion
    def _set_user_agent_header(self, user_agent_string: str = None) -> None:
```

I'm sure why `user_agent_string` has a default value -- that seems odd.  I left it there but you might consider removing it.